docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,243 @@
+:_content-type: PROCEDURE
+:myfunctionone: maintaining_operator_graphs
+
+[id="maintaining_operator_graphs_{context}"]
+= Maintaining an Operator graph
+
+File-based catalogs (FBC) provide a streamlined approach to store and distribute metadata about Operators written for Operator Lifecycle Manager (OLM). You can use {ProductName} to build, test, and release multiple components from a single FBC repo. 
+
+FBCs represent a departure from the previous method of defining operator graphs using a ClusterServiceVersion (CSV) file. Previously, operator developers defined operator graphs implicitly in CSV files. This made it difficult to maintain and update the graphs, as any changes required updating the CSV files and restarting the OLM controller.
+
+FBCs simplify graph management, offering enhanced control and ownership of operator graphs. With a choice of JSON or YAML formats, file-based catalogs ensure backward compatibility while optimizing efficiency.
+
+The advantages it offers are:
+
+* *Centralized graph updates:* FBC consolidates graph updates into a single catalog file, eliminating the need for iterative updates and simplifying the management of complex graphs.
+
+* *Convenient management and flexibility:* Compared to SQLite, FBC offers a more convenient solution for managing graph updates. Users can define all updates in one place using file-based catalogs, streamlining the process and improving overall efficiency. This centralized management capability simplifies the process and improves overall efficiency.
+
+* *Improved ownership and analysis:* FBC provides users with improved ownership and control over their graphs through clean and declarative templates for comprehensive analysis and surface level checks. This simplifies upgrade possibilities determination and error state recovery, eliminating the need to rely on limited configuration options found in localized CSV files.
+
+
+For additional information, see link:https://docs.openshift.com/container-platform/4.13/operators/understanding/olm-packaging-format.html#olm-file-based-catalogs_olm-packaging-format[File based Catalogs].
```

Comment:
Possible outline for changing this flow --

* Reword introduction to having a supported flow for producing an Operator catalog which defines the upgrade graph for an Operator
* Acknowledge that two different strategies exist, SQLite and FBC, and the difference between them
* State that AppStudio only supports the latter because of its benefits and since it has been supported since OCP 4.11.

@adellape, fyi